pkg/lib/size/size.go

```diff
@@ -1,18 +0,0 @@
-package size
```

Comment:
Where is the `Size` api now or has that been deprecated?

Reply:
Nothing was using it at all, the only user was the test.

Reply:
May want to check with @davidvossel or @adam because Size was added for one of them I believe..